packages/web3-shared/base/src/specs/index.ts

```diff
@@ -1146,13 +1148,12 @@ export interface TransactionFormatterState<ChainId, Parameters, Transaction> {
         chainId: ChainId,
         transaction: Transaction,
         context: TransactionContext<ChainId, Parameters>,
-        isError?: boolean
     ) => Promise<TransactionDescriptor<ChainId, Transaction>>
     /** Elaborate a transaction in a human-readable format. */
     formatTransaction: (
         chainId: ChainId,
         transaction: Transaction,
-        isError?: boolean
+        txHash?: string
```

Comment:
We can read a transaction hash from `Transaction.id`.

Reply:
There's no `id` property defined in `Transaction`.

Reply:
And why did the `isError`'s got removed?

Reply:
`isError` is added by me before, and it is useless.